vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -333,6 +339,13 @@ private String generatePodSpec(Map config) {
     podSpec.spec.containers = getContainerList(config)
     podSpec.spec.securityContext = getSecurityContext(config)
 
+    if (config.containerMountPath) {
+        podSpec.spec.volumes = [[
+                                    name    : "volume",
```

Comment:
If I understood it correctly, we are offering only one volume mount with fixed name "volume".  It means, user cannot come up with adding multiple volumes right?

Reply:
Yes, only one hardcoded volume for now. That is all we as CloudCI need.